### PR TITLE
[docs] Add supported libraries as tile style in Drivers and ORMs docs landing page

### PR DESCRIPTION
**Pull request by samiahmedsiddiqui**
@netlify /preview/drivers-orms/

### CODE REVIEW

**Comment by polarweasel** [comment]
(Updated description. The Netlify preview entry point will change with the next commit to this branch)

File: docs/content/preview/drivers-orms/_index.md [hunk 1]
```diff
@@ -29,20 +29,59 @@ Along with the smart drivers, YugabyteDB also supports upstream [PostgreSQL driv
...
+	<a href="csharp/" class="grid-child" title="C#">
+		<div>
+			<img src="/logos/csharp.svg" alt="C#" title="C#" />
+		</div>
+		C#
+	</a>
+	<a href="go/" class="grid-child" title="Go">
+		<div>
+			<img src="/logos/go.svg" alt="Go" title="Go" />
+		</div>
```
Comment (polarweasel) [inline docs/content/preview/drivers-orms/_index.md:42]: Will this work correctly here?
```suggestion
    <div class="icon">
      <i class="fa-brands fa-golang"></i>
    </div>
```

**Review by polarweasel — COMMENTED** [review]
Looking good... If it turns out Fontawesome won't support full-colour icons, then the static images are OK. We just went through some trouble to make better use of it, though, and remove some outdated logo images, so I'd rather use a library for this if possible...

**Comment by samiahmedsiddiqui** [comment]
@polarweasel Fontawesome doesn't support full-colour icons so that's why I have used static images. I know that it is a trouble to keep the icons up-to-date and Fontawesome also knows that, that's why they don't provide colour icons though.

**Review by polarweasel — APPROVED** [review]
I think this is a solid improvement over what's there now, so let's go with it.

**Pull request closed without merging**